src/webportal/src/app/user/fabric/userView/Table.jsx

```diff
@@ -0,0 +1,158 @@
+
+import Context from './Context';
+import Ordering from './Ordering';
+
+export default function Table() {
+  const {allUsers, filteredUsers, filter, ordering, setOrdering, pagination, setSelectedUsers, setAllSelected, editUser} = useContext(Context);
+
+  /**
+   * @type {import('office-ui-fabric-react').Selection}
+   */
+  const selection = useMemo(() => {
+    return new Selection({
+      onSelectionChanged() {
+        setSelectedUsers(selection.getSelection());
+        setAllSelected(selection.isAllSelected());
+      },
+    });
+  }, []);
+
+  /**
+   * @param {React.MouseEvent<HTMLElement>} event
+   * @param {import('office-ui-fabric-react').IColumn} column
+   */
+  function onColumnClick(event, column) {
+    const {field, descending} = ordering;
+    if (field === column.key) {
+      if (descending) {
+        setOrdering(new Ordering());
+      } else {
+        setOrdering(new Ordering(field, true));
+      }
+    } else {
+      setOrdering(new Ordering(column.key));
+    }
+  }
+
+  /**
+   * @param {import('office-ui-fabric-react').IColumn} column
+   */
+  function applySortProps(column) {
+    column.isSorted = ordering.field === column.key;
+    column.isSortedDescending = ordering.descending;
+    column.onColumnClick = onColumnClick;
+    return column;
+  }
+
+  /**
+   * @type {import('office-ui-fabric-react').IColumn}
+   */
+  const usernameColumn = applySortProps({
+    key: 'username',
+    minWidth: 200,
+    name: 'User Name',
+    fieldName: 'username',
+    className: FontClassNames.mediumPlus,
+    headerClassName: FontClassNames.medium,
+    isResizable: true,
+    isFiltered: filter.keyword !== '',
+  });
+
+  const adminColumn = applySortProps({
+    key: 'admin',
+    minWidth: 150,
+    name: 'Admin',
+    className: FontClassNames.mediumPlus,
+    headerClassName: FontClassNames.medium,
+    isResizable: true,
+    isFiltered: filter.admins.size > 0,
+    onRender(user) {
+      return toBool(user.admin) ? 'Yes' : 'No';
+    },
+  });
+
+  const virtualClusterColumn = applySortProps({
+    key: 'virtualCluster',
+    minWidth: 300,
+    name: 'Virtual Cluster',
+    className: FontClassNames.mediumPlus,
+    headerClassName: FontClassNames.medium,
+    isResizable: true,
+    isFiltered: filter.virtualClusters.size > 0,
+    onRender(user) {
+      return getVirtualCluster(user);
+    },
+  });
+
+  /**
+   * @type {import('office-ui-fabric-react').IColumn}
+   */
+  const actionsColumn = {
+    key: 'actions',
+    minWidth: 100,
+    name: 'Actions',
+    headerClassName: FontClassNames.medium,
+    columnActionsMode: ColumnActionsMode.disabled,
+    onRender(user) {
+      /**
+       * @param {React.MouseEvent} event
+       */
+      function onClick(event) {
+        event.stopPropagation();
+        editUser(user);
+      }
+      return (
+        <div className={c([t.dib, t.vMid, t.w100])} style={{margin: '-11px -8px -11px -12px'}} data-selection-disabled>
```

Comment:
You can use column's `className` to clear the default padding.

Reply:
Got it. Fixed.